src/ORM/FieldType/DBDate.php

```diff
@@ -271,18 +309,23 @@ public function Rfc822()
     public function Rfc2822()
     {
         if ($this->value) {
-            return date('Y-m-d H:i:s', strtotime($this->value));
+            return date('Y-m-d H:i:s', $this->getTimestamp());
```

Comment:
Fair enough, there's still around two dozen occurrences of `date()` in core - low priority, not a merge blocker.